Add new GSoC-2020 proposal for ATLAS geometry visualization

ric-bianchi opened the pull request:
Adding a new proposal for a visualization/detector-description/3d-graphics project in ATLAS.

aflp91 commented on _gsocproposals/2020/proposal_ATLAS-Visualization.md:
```diff
@@ -0,0 +1,60 @@
+---
+title: GeoModelExplorer, interactive 3D geometry visualization - Development of a visualization tool to interactively explore the  geometry of a HEP detector
+layout: gsoc_proposal
+project: ATLAS
```
Is there an ATLAS project described somewhere yet? If not, you have to add one there.

ric-bianchi replied:
Thanks a lot for your commet, @aflp91! 
I didn't notice that this was the first proposal as "ATLAS". 

I will add that now.

ric-bianchi replied:
Done.
Please, let me know if further additions or changes are needed.
Thanks, -- Riccardo.

ric-bianchi commented:
The CI tests failed, but I don't see errors triggered by my proposal page in the log. It looks like the errors are from links in other pages.

EdwardMoyse commented:
I saw the same for Phoenix.

xvallspl commented on _gsocprojects/2020/project_ATLAS.md:
```diff
@@ -0,0 +1,9 @@
+---
+project: ATLAS
+layout: default
+logo: ATLAS-Logo-Ref-RGB.png
```
The logo has to be named "ATLAS-logo.png"

ric-bianchi replied:
Done.

xvallspl commented on _gsocproposals/2020/proposal_ATLAS-Visualization.md:
All the mentors have to be in mentors.md

ric-bianchi replied:
Done.

ric-bianchi commented:
I added the missing entry to the global mentors' list and updated the filename of the ATLAS logo in the ATLAS project page, as requested.

Thanks for the corrections, and have a good day.

Best, -- Riccardo.
